sdk/tables/azure-data-tables/tests/test_table_entity.py

```diff
@@ -1221,6 +1221,29 @@ def test_delete_entity_overloads_kwargs(self, tables_storage_account_name, table
         finally:
             self._tear_down()
 
+    @tables_decorator
+    @recorded_by_proxy
+    def test_delete_entity_with_keys_in_uuid(self, tables_storage_account_name, tables_primary_storage_account_key):
+        # Arrange
+        self._set_up(tables_storage_account_name, tables_primary_storage_account_key)
+        try:
+            entity = {
+                u"PartitionKey": TEST_GUID,
+                u"RowKey": TEST_GUID,
+            }
+            self.table.upsert_entity(entity)
+            
+            result = self.table.get_entity(str(TEST_GUID), str(TEST_GUID))
```

Comment:
You shouldn't need the `str()` here if this is working correctly
```suggestion
            result = self.table.get_entity(TEST_GUID, TEST_GUID)
```